.github/workflows/valgrind-tests.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  workflow_call:
+    inputs:
+      portion:
+        required: true
+        type: string
+
+jobs:
+  valgrind_tests:
+    # name: Valgrind tests
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Is there something that prevents using checkout v3 here?

Reply:
no, just forgot about it since I moved that to a separate file